app/views/shared/_paginator.html.erb

```diff
@@ -0,0 +1,39 @@
+<div class="paginator">
+  <ol class="pagination">
+    <%=
+      content_tag :li, class: previous_link_css do
+        link_to bets_path(page: '1'), class: 'page-link' do
+          content_tag :i, nil, class: 'fa fa-angle-double-left'
+        end
+      end
+    %>
+    <%=
+      content_tag :li, class: previous_link_css do
+        link_to bets_path(page: current_page - 1), class: 'page-link' do
+          content_tag :i, nil, class: 'fa fa-angle-left'
+        end
+      end
+    %>
+
+    <% blocks.each do |page|%>
+      <%= render 'shared/blocks', page: page,
+        page_item_css: page_item_css(page,
+                                     current_page) %>
```

Comment:
This is kind of weird code style. You've decided to put 2 arguments on the first line and one on the second.